composer.json

```diff
@@ -18,32 +18,24 @@
     }
   ],
   "require": {
-    "php": "^7.1",
+    "php": "^7.3",
     "doctrine/annotations": "^1.6",
-    "sonata-project/admin-bundle": "^3.35",
-    "symfony/config": "^3.4 || ^4.0",
-    "symfony/dependency-injection": "^3.4 || ^4.0",
-    "symfony/finder": "^3.4 || ^4.0",
-    "symfony/http-kernel": "^3.4 || ^4.0"
-  },
-  "conflict": {
-    "sonata-project/admin-bundle": "<3.35"
+    "sonata-project/admin-bundle": "dev-master",
```

Comment:
why is it like? don't think we can release it like this, because this is unstable

Reply:
Yes, we cannot release until sonata release theirs v4 versions which will be the version Symfony 5 compatible.